### PR TITLE
Replicate initialization from tensorflow

### PR DESCRIPTION
**Pull request by kmatzen**
There are some minor inconsistencies in network initialization with respect to the TensorFlow implementation.  In particular, tf.keras.layers.Dense will use a Glorat (Xavier in PyTorch) uniform initialization scaled based on the linear layer's non-linearity and will zero out the bias.  However, torch.nn.Linear will always use Kaiming (He in TensorFlow) uniform initialization scaled with the assumption that a leaky ReLU non-linearity with negative slope = sqrt(5) will be used.  It also uses a uniform bias initialization.

The impact of using the wrong initialization is that activations throughout the network will significantly change in magnitude. 
 This is exacerbated if the depth of the network is increased resulting in vanishing gradients.  Switching the weight initialization to at least match the correct non-linearity being used will result in consistent activation magnitude throughout the network.

I ran experiments using the blender lego scene to verify that the change didn't significantly degrade previously established quality.  On average, the test set before the change had a PSNR of 31.47 and after had a PSNR of 31.52.

https://www.tensorflow.org/api_docs/python/tf/keras/layers/Dense
```
tf.keras.layers.Dense(
    units, activation=None, use_bias=True,
    kernel_initializer='glorot_uniform',
    bias_initializer='zeros', kernel_regularizer=None,
    bias_regularizer=None, activity_regularizer=None, kernel_constraint=None,
    bias_constraint=None, **kwargs
)
```

https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/linear.py#L86-L91
```
    def reset_parameters(self) -> None:
        init.kaiming_uniform_(self.weight, a=math.sqrt(5))
        if self.bias is not None:
            fan_in, _ = init._calculate_fan_in_and_fan_out(self.weight)
            bound = 1 / math.sqrt(fan_in)
            init.uniform_(self.bias, -bound, bound)
```

### CODE REVIEW

**Comment by yenchenlin** [comment]
Hello @kmatzen , huge thanks for finding it out and verifying the results through experiments.
